kayenta-core/src/main/java/com/netflix/kayenta/storage/ObjectType.java

```diff
@@ -31,7 +32,9 @@
   CANARY_CONFIG(new TypeReference<CanaryConfig>() {}, "canary_config", "canary_config.json"),
   METRIC_SET_LIST(new TypeReference<List<MetricSet>>() {}, "metrics", "metric_sets.json"),
   METRIC_SET_PAIR_LIST(new TypeReference<List<MetricSetPair>>() {}, "metric_pairs", "metric_set_pairs.json"),
-  CANARY_RESULT(new TypeReference<CanaryResult>() {}, "canary_results", "canary_results.json");
+  CANARY_RESULT_ARCHIVE(new TypeReference<CanaryExecutionStatusResponse>() {}, "canary_archive", "canary_archive.json"),
```

Comment:
We don't have a proper sort order here, but I _think_ I would have put `CANARY_RESULT_ARCHIVE` after `CANARY_RESULT` for reasons I can't really explain.

Reply:
Heh.  OK.  :)